packages/components/src/components/ScalarSearchResults/ScalarSearchResultItem.vue

```diff
@@ -12,9 +12,10 @@ defineOptions({ inheritAttrs: false })
 const { cx } = useBindCx()
 </script>
 <template>
-  <li
+  <a
```

Comment:
would set `a` within `li` vs replace it achieve the same? so it keeps the ul > li markup relationship

Reply:
Updated!

Reply:
yes make sense! was just a nit pick but not against ul > div then!